test/alerts/parser_test.exs

```diff
@@ -58,15 +58,22 @@ defmodule Alerts.ParserTest do
                      }
                    ]),
                  active_period: [
-                   {~N[2016-06-06T14:48:48] |> Timex.to_datetime("Etc/GMT+4"),
-                    ~N[2016-06-06T19:53:51] |> Timex.to_datetime("Etc/GMT+4")}
+                   {~N[2016-06-06T14:48:48]
+                    |> Timex.to_datetime("Etc/GMT+4")
+                    |> Timex.Timezone.convert("America/New_York"),
```

Comment:
This fixes the failing test. The code does this timezone conversions, but the tests weren't changed to also do it.